client/src/app/components/post_creation/posts.component.html

```diff
@@ -1,73 +1,92 @@
-<form (ngSubmit)="onSubmit()">
-  <label> Post Title: <input type="text" [(ngModel)]="title" name="title" required>
-  </label>
-  <br>
-  <label> Post Image:<input type="file" accept="image/*" id="imageCapturer" (change)="onFileSelected($event)">
-  </label>
-  <br>
-
-  <label id="requirement-selection-label">Select the requirement creation medium</label>
-  <br>
-  <mat-radio-group [(ngModel)]="requirementsType" name="requirementSelection">
-    <mat-radio-button [value]="0">Requirements through the website</mat-radio-button>
-    <mat-radio-button [value]="1">Requirements through a file</mat-radio-button>
-    <mat-radio-button [value]="2">Requirements through the website and a file</mat-radio-button>
-  </mat-radio-group>
-  <br><br>
-
-  <div id="websiteReqs" *ngIf="requirementsType == 0">
-
-    <label> Description: <input type="text" [(ngModel)]="description" name="description" required>
+<main>
+  <form (ngSubmit)="onSubmit()">
+    <label> 
+      Post Title: <br>
+      <input type="text" [(ngModel)]="title" name="title" required>
+      <br>
     </label>
-    <br />
-    <label> Responsibilities: <input type="text" [(ngModel)]="responsibilities" name="responsibilities" required>
+    <br>
+    <label> 
+      Post Image: <br>
+      <input type="file" accept="image/*" id="imageCapturer" (change)="onFileSelected($event)">
+      <br>
     </label>
-    <br />
+    <br>
 
-    <p>Research Category<span
-        matTooltip="Research categories are used to recommend opportunites to relevant students and for searching"
-        matTooltipClass="example-tooltip-uppercase"><i class="material-icons">help_outline</i></span></p>
-    <ng-container #categories></ng-container>
+    <label id="requirement-selection-label">Select the requirement creation medium</label>
     <br>
-    <input type="button" value="Add New Research Category" (click)="createNewCategory('', false)">
+    <mat-radio-group [(ngModel)]="requirementsType" name="requirementSelection">
+      <mat-radio-button [value]="0">Requirements through the website</mat-radio-button> <br>
+      <mat-radio-button [value]="1">Requirements through a file</mat-radio-button> <br>
+      <mat-radio-button [value]="2">Requirements through the website and a file</mat-radio-button> <br>
+    </mat-radio-group>
     <br><br>
 
-    <label> GPA Requirement: <input type="number" [(ngModel)]="gpa" name="gpa" min="0" max="4">
-    </label><br>
-    <label> Major(s):</label>
-    <br>
-    <ng-container #majors></ng-container>
-    <input type="button" value="Add New Major" (click)="createNewCategory('', true)">
-    <br>
-    <br>
-    <label> Deadline: <mat-form-field color="accent" appearance="fill">
+    <div id="websiteReqs" *ngIf="requirementsType == 0">
+
+      <label> 
+        Description: <br> 
+        <input class = "text-field" type="text" [(ngModel)]="description" name="description" required>
+        <br>
+      </label>
+      <br />
```

Comment:
Try to be consistent with the `<br>` tags - use only `<br>`, as that is what's used in other components.